email_sender/pkg/utils/serializer/json.go

```diff
@@ -164,3 +151,52 @@ func (e *ForbiddenError) Error() string {
 func (e *IsConflictError) Error() string {
 	return e.msg
 }
+
+// swagger:model message
+type JsonMessage struct {
+	Message string `json:"message"`
+}
+
+func SendJSON(w http.ResponseWriter, status int, object interface{}) error {
+	// set headers
+	SetCorsHeaders(w)
+	w.Header().Set("Content-Type", "application/json")
+	// write response
+	b, err := json.Marshal(object)
```

Comment:
can we try to use here json.NewEncoder(w).Encode(object)

Reply:
yes, that's right, thank you